app/views/upload_your_files.scala.html

```diff
@@ -78,16 +78,15 @@
         }.map(msg => <p class="govuk-body"><strong>{msg}</strong></p>)
     }
 
-    @formHelper(Call("POST", uploadRequest.href), args = Symbol("enctype") -> "multipart/form-data") {
-
+    <form action="@uploadRequest.href" method="POST" enctype="multipart/form-data">
         <div class="govuk-form-group">
             @uploadRequest.fields.map { case (key, value) =>
                 <input type="hidden" name="@key" value="@value"/>
             }
 
             @govukFileUpload(FileUploadComponent(
                 id = fileUploadId,
-                name = fileUploadId,
+                name = "file",
```

Comment:
Uhm... Just worried for [this JS line](https://github.com/hmrc/cds-file-upload-frontend/blob/00ba68d9e471b65001e0c6b2e90d015229fca397/app/assets/javascripts/validation.js#L12)
But I supposed you have tried your change and the validation is still working, isn't it?